src/openrct2/Version.cpp

```diff
@@ -9,17 +9,30 @@
 
 #include "Version.h"
 
-#include <string>
+#include "config/Config.h"
+#include "core/Console.hpp"
+#include "core/Http.h"
+#include "core/Json.hpp"
+
+#include <chrono>
 
 #ifdef OPENRCT2_BUILD_INFO_HEADER
 #    include OPENRCT2_BUILD_INFO_HEADER
 #endif
 
+const char gVersionInfoTag[] =
```

Comment:
I made some tests by spoofing this value

Reply:
I tried to test this PR by changing this line to read:
```diff
-const char gVersionInfoTag[] =
+const char gVersionInfoTag[] = "v0.2.5";
```

But I'm not getting a popup on the title screen. I made sure to delete the `last_version_check_time` from `config.ini` before testing.

Any ideas?

Reply:
I did test it earlier with such change and seemed ok. That was few revisions ago, so I'll recheck